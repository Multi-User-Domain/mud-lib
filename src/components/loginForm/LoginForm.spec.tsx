import React from "react";
import { render } from "@testing-library/react";

import { LoginForm } from "./LoginForm";

it("renders", () => {
  const { asFragment } = render(<LoginForm />);
  expect(asFragment()).toMatchSnapshot();
});
