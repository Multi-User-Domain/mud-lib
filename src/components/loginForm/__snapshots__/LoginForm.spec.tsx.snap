// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="chakra-container css-0"
    fixed="true"
  >
    <div
      class="chakra-input__group css-4302v8"
    >
      <input
        class="chakra-input css-tsqr3a"
        label="Identity Provider"
        placeholder="Identity Provider"
        type="url"
        value="https://inrupt.net"
      />
      <div
        class="chakra-input__right-addon css-nxc9bd"
      >
        <div
          role="button"
          tabindex="0"
        >
          <button
            class="chakra-button css-pvkd4k"
            type="button"
          >
            LogÂ in
          </button>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
